src/service/mercury/helpers/transformers.ts

```diff
@@ -75,21 +78,21 @@ const transformAccountBalances = async (
   const classicBalanceData = rawResponse?.data?.balanceByPublicKey.nodes || [];
 
   const accountObject = accountObjectData[0];
-  // TODO: get these into query
-  const numSponsoring = 0;
-  const numSponsored = 0;
-  const sellingLiabilities = 0;
+  const numSubEntries = accountObject.numSubEntries || "0";
```

Comment:
unrelated, Mercury has now added these fields to the account object so added them here.